Resources/Private/Templates/Backend/Index.html

```diff
@@ -3,6 +3,7 @@
 <html>
     <head>
         <title>Neos CMS</title>
+        <meta name="viewport" value="width=device-width">
```

Comment:
I think this should be `<meta name="viewport" content="width=device-width, initial-scale=1.0">`